setup.py

```diff
@@ -26,9 +26,7 @@
         "typing-extensions>=4.2.0,<5.0.0",
     ],
     extras_require={
-        "hiredis-async": ["aioredis[hiredis]>=2.0.1,<3.0.0"],
```

Comment:
Should the hiredis opt-in option still be one of the extras? e.g. add 
```
"hiredis": ["redis[hiredis]>=4.3.0,<5.0.0"],
```

Reply:
I had my doubts about that and decided that exposing the implementation flags isn't quite important here. `redis-py` uses `hiredis` automatically when the latter is available – one always can `hiredis` as an explicit dependency.

Reply:
In that case, that would be worth adding to the README imo. Maybe better even to recommend setting `redis[hiredis]` as an explicit dependency to ensure compatibility between redis-py & hiredis. I can imagine a scenario for instance where a `hiredis` dependency gets auto-updated to 2.x and subsequently `redis-py` stops using `hiredis` under the hood